src/components/ShopList/ShopList.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+
+import ShopItem from './Components/ShopItem'
+import ShopFilter from '../../pages/Shop/Components/ShopFilter/ShopFilter'
+
+import {PRODUCT_LIST} from '../.././config'
+import './ShopList.scss'
+
+class ShopList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      shopListData: [],
+      filterValue: 'recent'
+    }
+  }
+
+  componentDidMount() {
+    fetch(PRODUCT_LIST)
+      .then((res) => res.json())
+      .then((data) => {
+        this.setState({
+          shopListData: data["product_list"]
+        })
+      })
+  }
+
+  componentDidUpdate() {
+    const {filterValue} = this.state;
+    fetch(`PRODUCT_LIST?ordering=${filterValue}`)
+      .then((res) => res.json())
+      .then((data) => {
+        console.log(data)
+      })
+  }
+
+  selectFilter = (evt) => {
+    this.setState({
+      filterValue: evt.target.value
+    })
+  }
+
+
+  render() {
+    const {shopListData, filterValue} = this.state;
+
+    return(
+      <>
+        <ShopFilter
+        selectFilter={this.selectFilter}
+        />
+        <ul className="ShopList">
+        {shopListData && shopListData.map((data) => {
+          return(
+            <ShopItem
+            key={data["product_id"]} 
```

Comment:
```suggestion
            key={data.product_id} 
```
계산된 속성명을 쓰지 않을 경우라면 dot notation을 사용해주세요.

Reply:
넵~~